### PR TITLE
Update article.md

### PR DESCRIPTION
**Pull request by Zearin**
Minor edits for grammar, phrasing, and markup.

### CODE REVIEW

**Comment by iliakan** [comment]
Thanks, there should be changes though.
-We don't use `<dfn>` tag.
- Italic is `*...*`, not `_..._` =)

File: 2-ui/4-forms-controls/1-form-elements/article.md [hunk 1]
```diff
@@ -182,7 +182,7 @@ A `<select>` element has 3 important properties:
 
 They provide three different ways of setting a value for a `<select>`:
 
-1. Find the corresponding `<option>` element (e.g. among `select.options`) and set its `option.selected` to `true`.
+1. Find the corresponding `<option>` element (e.g., among `select.options`) and set its `option.selected` to `true`.
```
Comment (iliakan) [inline 2-ui/4-forms-controls/1-form-elements/article.md:185]: A comma here? Why?

File: 2-ui/4-forms-controls/1-form-elements/article.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ Working with forms will be much more convenient when we learn them.
 
 Document forms are members of the special collection `document.forms`.
 
-That's a so-called "named collection": it's both named and ordered. We can use both the name or the number in the document to get the form.
+That's a so-called **"named collection"**: it's both named and ordered. We can use both the name or the number in the document to get the form.
```
Comment (iliakan) [inline 2-ui/4-forms-controls/1-form-elements/article.md:11]: You make it bold this way. Are you sure? Maybe should be italic (single `*`)?

File: 2-ui/4-forms-controls/1-form-elements/article.md [hunk 3]
```diff
@@ -246,7 +246,7 @@ This syntax is optional. We can use `document.createElement('option')` and set a
 
 The difference between `defaultSelected` and `selected` is that `defaultSelected` sets the HTML-attribute (that we can get using `option.getAttribute('selected')`, while `selected` sets whether the option is selected or not.
 
-In practice, we usually should set both values to `true` or `false` (or omit, that's the same as `false`).
+In practice, you should usually set _both_ values to `true` or `false` (or omit; that's the same as `false`).
```
Comment (iliakan) [inline 2-ui/4-forms-controls/1-form-elements/article.md:249]: you → one.
And please no `_`. This formatting is never used in the tutorial.

Also are sure about ; ?

File: 2-ui/4-forms-controls/1-form-elements/article.md [hunk 4]
```diff
@@ -289,9 +289,9 @@ Form navigation:
 `element.form`
 : Elements reference their form in the `form` property.
 
-Value is available as `input.value`, `textarea.value`, `select.value` etc, or `input.checked` for checkboxes and radio buttons.
+Value is available as `input.value`, `textarea.value`, `select.value`, etc. (or `input.checked` for checkboxes and radio buttons).
```
Comment (iliakan) [inline 2-ui/4-forms-controls/1-form-elements/article.md:292]: Are you sure the dot is correct at the end? Doesn't look so.

**Review by iliakan — CHANGES_REQUESTED** [review]
Are you a native English speaker? Just for me to know ;)

File: 2-ui/4-forms-controls/1-form-elements/article.md (hunk 1 above)
Comment (Zearin) [reply 2-ui/4-forms-controls/1-form-elements/article.md:185]: It is habit from a practice I've maintained.  The TL;DR of it is, a comma should follow an “e.g.”—but _not_ an “i.e.”.

Do you feel strongly about it?

File: 2-ui/4-forms-controls/1-form-elements/article.md (hunk 2 above)
Comment (Zearin) [reply 2-ui/4-forms-controls/1-form-elements/article.md]: Done.

File: 2-ui/4-forms-controls/1-form-elements/article.md (hunk 3 above)
Comment (Zearin) [reply 2-ui/4-forms-controls/1-form-elements/article.md:249]: I am not sure about the semi-colon.  But I *am* sure that it should definitely not be a comma.  That is a comma-splice. :)

File: 2-ui/4-forms-controls/1-form-elements/article.md (hunk 4 above)
Comment (Zearin) [reply 2-ui/4-forms-controls/1-form-elements/article.md:292]: I'll rewrite...  Just a moment.

Comment (Zearin) [reply 2-ui/4-forms-controls/1-form-elements/article.md]: Rewritten with less weird punctuation at the endpoints. :)

File: 2-ui/4-forms-controls/1-form-elements/article.md (hunk 3 above)
Comment (Zearin) [reply 2-ui/4-forms-controls/1-form-elements/article.md]: Changed “you” to “one”.  Rewrote the statement about omission.

**Comment by Zearin** [comment]
> Are you a native English speaker? Just for me to know ;)

(Sorry for the late reply!  I missed this comment.)  

✔ Yes, I am a native English speaker.

File: 2-ui/4-forms-controls/1-form-elements/article.md (hunk 1 above)
Comment (iliakan) [reply 2-ui/4-forms-controls/1-form-elements/article.md:185]: I almost never see e.g. with a comma. I'm not sure it's correct. Can you please remove the comma?

**Comment by iliakan** [comment]
Ok, please no comma after e.g. =)

**Comment by iliakan** [comment]
Ping @Zearin ? ;)

**Comment by Zearin** [comment]
> Ping @Zearin ? ;)

Sorry about that—and thanks for the ping!  

✅ Comma removed.

File: 2-ui/4-forms-controls/1-form-elements/article.md (hunk 1 above)
Comment (Zearin) [reply 2-ui/4-forms-controls/1-form-elements/article.md]: Done!

**Review by Sebastianokunkelo — COMMENTED** [review]
Zearin:patch-1

**Pull request merged**